src/NzbDrone.Core/Indexers/HttpIndexerBase.cs

```diff
@@ -470,6 +472,12 @@ protected virtual async Task<IndexerPageableQueryResult> FetchReleases(Func<IInd
                 {
                     _logger.Warn(webException, "{0} {1} {2}", this, url, webException.Message);
                 }
+
+                if (lastRequest != null && lastRequest.HttpRequest != null)
```

Comment:
Happy to shift this to only apply to timeouts, but overall this will recycle the client anytime there is a failure, which would have put it into the state where it is backing off with further and further away so it shouldn't impact perf too much.